public/data/akshithbellare.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "Akshith Bellare",
+  "type": "personal",
+  "bio": ".NET Core Developer and Open Source Enthusiast!",
+  "avatar": "https://github.com/AKSHITHBELLARE.png",
+  "links": [
+    {
+      "name": "My Personal Website 🌐",
+      "url": "https://akshithbellare.github.io",
+      "icon": "link"
+    },
+    {
+      "name": "Follow me on GitHub ❤️",
+      "url": "https://github.com/AKSHITHBELLARE",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter 🗨️",
+      "url": "https://twitter.com/AKSHITHBELLARE",
+      "icon": "twitter"
+    },
+    {
+      "name": "Connect with me on LinkedIn 🤝",
+      "url": "https://www.linkedin.com/in/AKSHITHBELLARE",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Go to my Youtube! 🎥",
```

Comment:
I recommend capitalizing the `t` here. What do you think?
```suggestion
      "name": "Go to my YouTube! 🎥",
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inelegant